src/prefect/utilities/storage.py

```diff
@@ -82,3 +83,35 @@ def extract_flow_from_file(
                 return exec_vals[var]
 
     raise ValueError("No flow found in file.")
+
+def extract_flow_from_module(
+    module_str: str, flow_name: str = None
+) -> "Flow":
+    """
+    Extract a flow object from a python module.
+
+    Args:
+        - module_str (str, optional): A module path pointing to a .py file containing a flow.
+            For example, 'myrepo.mymodule.myflow' where myflow.py contains the flow.
+        - flow_name (str, optional): A specific name of a flow to extract from a file.
+            If not set then the first flow object retrieved from file will be returned.
+            The "first" flow object will be based on the dir(module) ordering, which
+            is alphabetical and capitalized letters come first.
+
+    Returns:
+        - Flow: A flow object extracted from a file
+    """
+
+    # load the module
+    module = importlib.import_module('prefecttest.flow')
```

Comment:
Looks like some testing hardcoded values - probably meant `module_str` here?

Reply:
oh wow, I can't believe I missed this. Thanks for catching it. Yes, it's supposed to be module_str!